docs/android.md

```diff
@@ -233,48 +297,59 @@ the [ABI/architecture table](#abi-table)).
 The exact list of libraries to include depend on what features you're using but
 the basic list of library file for OpenDDS are as follows:
 
- - If not already included because of a separate C++ NDK project, you must
-   include the Clang C++ Standard Library. This is located at
-   `$TOOLCHAIN/sysroot/usr/lib/$ABI_PREFIX/libc++_shared.so` where
-   `$ABI_PREFIX` is an identifier for the architecture and can be found in the
-   [ABI/architecture table](#abi-table).
+  - Required Libraries to use OpenDDS
+
+   - If not already included because of a separate C++ NDK project, you must
+     include the Clang C++ Standard Library. This is located at
+     `$TOOLCHAIN/sysroot/usr/lib/$ABI_PREFIX/libc++_shared.so` where
+     `$ABI_PREFIX` is an identifier for the architecture and can be found in the
+     [ABI/architecture table](#abi-table).
+
+   - `$ACE_ROOT/lib/libACE.so`
+   - `$ACE_ROOT/lib/libTAO.so`
+   - `$ACE_ROOT/lib/libTAO_AnyTypeCode.so`
+   - `$ACE_ROOT/lib/libTAO_BiDirGIOP.so`
+   - `$ACE_ROOT/lib/libTAO_CodecFactory.so`
+   - `$ACE_ROOT/lib/libTAO_PI.so`
 
- - `$DDS_ROOT/lib/libOpenDDS_Dcps.so`
- - `$ACE_ROOT/lib/libACE.so`
- - `$ACE_ROOT/lib/libTAO.so`
- - `$ACE_ROOT/lib/libTAO_AnyTypeCode.so`
- - `$ACE_ROOT/lib/libTAO_BiDirGIOP.so`
- - `$ACE_ROOT/lib/libTAO_CodecFactory.so`
- - `$ACE_ROOT/lib/libTAO_PI.so`
+   - `$DDS_ROOT/lib/libOpenDDS_Dcps.so`
 
  - The following are the transport libraries, one for each transport type. You
    will need at least one of these, depending on the transport(s) you want to
    use:
    - `$DDS_ROOT/lib/libOpenDDS_Rtps_Udp.so`
-     - You will also need `$DDS_ROOT/lib/libOpenDDS_Rtps.so`
+     - Depends on `$DDS_ROOT/lib/libOpenDDS_Rtps.so`.
    - `$DDS_ROOT/lib/libOpenDDS_Multicast.so`
    - `$DDS_ROOT/lib/libOpenDDS_Shmem.so`
    - `$DDS_ROOT/lib/libOpenDDS_Tcp.so`
    - `$DDS_ROOT/lib/libOpenDDS_Udp.so`
 
- - The [type support library for your IDL](#cross-compiling-idl-libraries)
+ - The [type support libraries for your IDL](#cross-compiling-idl-libraries)
 
- - Required to use InfoRepo Peer Discovery:
-   - `$DDS_ROOT/lib/libOpenDDS_InfoRepoDiscovery.so`
-   - `$ACE_ROOT/lib/libTAO_PortableServer.so`
+ - The following are the peer discovery libraries. Static discovery is built
+   into `libOpenDDS_Dcps.so`, but most likely you will want one of these:
+
+   - Required to use RTPS Peer Discovery:
+     - `$DDS_ROOT/lib/libOpenDDS_Rtps.so`
+
+   - Required to use the Information Repository Peer Discovery:
+     - `$DDS_ROOT/lib/libOpenDDS_InfoRepoDiscovery.so`
+       - Depends on `$ACE_ROOT/lib/libTAO_PortableServer.so`
 
  - Required to use OpenDDS Security:
    - `$ACE_ROOT/lib/libACE_XML_Utils.so`
    - `$DDS_ROOT/lib/libOpenDDS_QOS_XML_XSC_Handler.so`
```

Comment:
QOS_XML_XSC is not required for OpenDDS Security

Reply:
Yes, that makes sense now. I don't think I knew what it was at the time, I probably just saw it was in the `lib` directory.